x-aeson/ambiata-x-aeson.cabal

```diff
@@ -15,7 +15,11 @@ library
                        aeson                           >= 0.8        && < 0.12
                      , base                            >= 3          && < 6
                      , bytestring                      == 0.10.*
+                     , containers                      == 0.5.*
                      , text                            >= 1.1        && < 1.3
+                     , time                            >= 1.5        && < 2
```

Comment:
Fair enough, maybe we should fix that (`tz` / `tzdata`), `time` is a bit of a disaster zone, lazy `Integer`s everywhere. It just can't be used for performance sensitive code.